Resources/Prototypes/_SpaceCats/Entities/Objects/Devices/cartridges.yml

```
@@ -0,0 +1,20 @@
+    state: cart-chat
+  - type: UIFragment
+    ui: !type:NanoChatUi
+  - type: NanoChatCartridge
+  - type: Cartridge
+    programName: nano-chat-program-name
+    icon:
+      sprite: _SpaceCats//Misc/program_icons.rsi
+      state: nanochat
+  - type: ActiveRadio
+    channels:
+    - Common
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)